### PR TITLE
Vg

### PR DESCRIPTION
**Pull request by J0WI**
this needs some testing, especially video (Flash) content

### CODE REVIEW

**Comment by J0WI** [comment]
*bump*

File: src/chrome/content/rules/VG.no.xml [hunk 1]
```diff
@@ -1,22 +1,36 @@
...
 	<target host="static04.vg.no" />
-	<rule from="^http://www\.vg\.no/" to="https://www.vg.no/"/>
-	<rule from="^http://vg\.no/" to="https://vg.no/"/>
-	<rule from="^http://static\.vg\.no/" to="https://static.vg.no/"/>
-	<rule from="^http://static01\.vg\.no/" to="https://static01.vg.no/"/>
-	<rule from="^http://static02\.vg\.no/" to="https://static02.vg.no/"/>
-	<rule from="^http://static03\.vg\.no/" to="https://static03.vg.no/"/>
-	<rule from="^http://static04\.vg\.no/" to="https://static04.vg.no/"/>
-</ruleset>
+	<target host="touch.vg.no" />
```
Comment (TheNavigat) [inline src/chrome/content/rules/VG.no.xml]: This subdomain redirects to ```https://touch.vg.no``` which redirect to ```http://www.vg.no```. This target host basically achieves nothing.

**Comment by J0WI** [comment]
cc @fuglede @reedy @loveithateit

File: src/chrome/content/rules/VG.no.xml (hunk 1 above)
Comment (semenko) [inline src/chrome/content/rules/VG.no.xml]: No downside to securing it -- and perhaps it has specific URLs that do magic things

**Pull request merged**